src/cascadia/TerminalCore/TerminalSelection.cpp

```diff
@@ -82,6 +82,24 @@ const til::point Terminal::GetSelectionEnd() const noexcept
     return _selection->end;
 }
 
+til::point Terminal::SelectionStartForRendering() const
+{
+    auto pos{ _selection->start };
+    const auto bufferSize{ GetTextBuffer().GetSize() };
+    bufferSize.DecrementInBounds(pos);
+    pos.Y = base::ClampSub(pos.Y, _VisibleStartIndex());
+    return til::point{ pos };
+}
+
+til::point Terminal::SelectionEndForRendering() const
+{
+    auto pos{ _selection->end };
+    const auto bufferSize{ GetTextBuffer().GetSize() };
+    bufferSize.IncrementInBounds(pos);
+    pos.Y = base::ClampSub(pos.Y, _VisibleStartIndex());
+    return til::point{ pos };
+}

```

Comment:
please doc these. It's especially important because **it's not used by the renderer**...